doc/source/_static/version_switcher.json

```diff
@@ -0,0 +1,140 @@
+[
+    {
+        "name": "1.9.0 (dev)",
+        "version": "dev"
+    },
+    {
+        "name": "1.8.0 (rc)",
+        "version": "1.8.0"
+    },
+    {
+        "name": "1.7.3 (stable)",
+        "version": "1.7.3"
+    },
+    {
+        "version": "1.7.2"
+    },
+    {
+        "version": "1.7.1"
+    },
+    {
+        "version": "1.7.0"
+    },
```

Comment:
These will depends on how we fix the deployment of the live docs. Previous versions' links are resolving properly.